recipes/xoshiro-cpp/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class XoshiroCppConan(ConanFile):
+    name = "xoshiro-cpp"
+    description = "Header-only Xoshiro/Xoroshiro PRNG wrapper library for modern C++ (C++17/C++20)"
+    license = "MIT"
+    homepage = "https://github.com/Reputeless/Xoshiro-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("prng", "xoshiro", "header-only")
+    settings = "arch", "build_type", "compiler", "os"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
```suggestion
    generators = "cmake"
```
Because your recipe doesn't require any recipes, cmake_find_package_multi is not necessary.

Please refer following:
https://github.com/conan-io/conan-center-index/blob/master/docs/package/all/conanfile.py#L26-L30